text/0000-sawtooth-governance.md

```diff
@@ -0,0 +1,348 @@
+the consensus interface
+
+**Seth** - Responsible for the Sawtooth integration with Ethereum
+
+**Community Outreach** - Responsible for writing and hosting documentation,
+managing the main Sawtooth website, promoting Sawtooth to the public, managing
+demos, and providing the community with training and support
+
+**Release Management** - Responsible for release related issues such as
+dependency management, license compliance, version control, and upgrade and
+backwards compatibility
+
+**Continuous Integration** - Responsible for test and build environments and
+deployment artifacts
+
+## Contributor Permission Levels
+
+The Sawtooth community encourages contributions to the project from all
+interested individuals. The project defines three levels of permissions that
+determine what actions a contributor can perform on a particular repository.
+
+A **reviewer** has "read" permission, meaning they can review pull requests.
+
+A **committer** has "write" permission, meaning they can merge pull requests
+once they have been approved.
+
+Any community member can be promoted to a reviewer or committer by a
+maintainer. Reviewers and committers lose their status when it is removed by a
+maintainer. This is usually due to inactivity, but can also be used for
+moderation.
+
+A **maintainer** has permission to approve changes. All pull-requests must be
+approved by at least 2 maintainers before a committer may merge it.
+Maintainers are expected to:
+
+- Carefully review pull requests and leave thoughtful, constructive feedback
+- Ensure the component is up-to-date with patches and security updates
+- Participate in community discussions related to their component
+
+A committer can be promoted to a maintainer by unanimous vote by the existing
+maintainers. Maintainers lose their status due to:
+
+- Inactivity of 12 Months
+- Unanimous vote by all other Maintainers
+
+## Code of Conduct
+
+Members of the Sawtooth community are expected to abide by the [Hyperledger Code of Conduct][hyperledger-coc].
+Violations of this code of conduct by any community member may result in
+disciplinary action at the discretion of the root team (by unanimous vote,
+excluding the offending member if applicable).
+
+[hyperledger-coc]: https://wiki.hyperledger.org/community/hyperledger-project-code-of-conduct
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+This RFC makes explicit a governance model. Since this was previously implicit,
+some existing community members may feel offended by not being given a
+permission level that is consistent with their expectations or by not being
+placed on a team whose domain they feel responsible for.
+
+# Rationale and alternatives
+[alternatives]: #alternatives
+
+The [Mozilla governance model](https://www.mozilla.org/en-US/about/governance/policies/module-ownership/ )
+was also considered, however a hierarchical model was focused on ensuring the
+project maintained a consistent vision was preferred.
```

Comment:
Also http://www.apache.org/foundation/governance/pmcs.html which describes the ASF model.

Reply:
Thanks for the link